app/Forms/Builders/UserForm.php

```diff
@@ -23,6 +23,7 @@ public function create()
 
     public function edit(User $user)
     {
+        unset($user->password);
```

Comment:
We don't need that, the *password* field is in the `$guarded` array for the `User` model, so there is nothing to unset.